plugins/LOMM/LOMM.cpp

```diff
@@ -97,6 +93,11 @@ void LOMMEffect::changeSampleRate()
 			m_scLookBuf[j][i].resize(m_lookBufLength, LOMM_MIN_FLOOR);
```

Comment:
Is it wanted that `m_inLookBuf` is being initialized to `0.f`, while the other arrays are initialized to `LOMM_MIN_FLOOR`?

Reply:
Yes.  The lookahead buffer stores past audio.  Many of these other variables are used as denominators and shouldn't ever be zero.